### PR TITLE
enable support for multiple socketpaths for docker 1.8 compatability

### PR DESCRIPTION
**Pull request by binocarlos**
upon booting - loop over a list of socketpaths and create the directory (if it does not exist)

use the same adapter and application and mount the adapter on each of the socket paths

### CODE REVIEW

File: flockerdockerplugin/flockerdockerplugin.tac [hunk 1]
```diff
@@ -18,7 +33,27 @@ def getAdapter():
...
-adapterServer = internet.UNIXServer("/usr/share/docker/plugins/flocker.sock", getAdapter())
-adapterServer.setServiceParent(application)
+"""
+Loop over each socketPath - create the folder and then mount the adapter
+on the socket for that path
+"""
+for socketPath in socketPaths:
+    dirName = os.path.dirname(socketPath)
+    if not os.path.exists(dirName):
+        os.makedirs(dirName)
```
Comment (robhaswell) [inline flockerdockerplugin/flockerdockerplugin.tac:58]: You should manually set the permission on this directory after creating it.

**Comment by robhaswell** [comment]
OK. I don't think this is the right strategy - listening on multiple sockets is error-prone. We should just listen on the new socket. However, this is all Labs stuff so it's probably not worth agonising over. The full version won't have this problem.

Please don't bump version numbers in feature branches - they should be separate PRs.

**Comment by robhaswell** [comment]
LGTM, merge when you want.

**Comment by binocarlos** [comment]
I've removed the version bump from this PR and made a [separate PR](https://github.com/ClusterHQ/flocker-docker-plugin/pull/18) for it

Merging - thanks!

**Pull request merged**